CHANGELOG.md

```diff
@@ -30,6 +30,12 @@ You can check your current version with the following command:
 
 For more information, see [UP42 Python package description](https://pypi.org/project/up42-py/).
 
+## 0.37.0a0
```

Comment:
check the order of merging as the other PR has the same version bump

Reply:
I'll address that in my PR if this is merged earlier

Reply:
I updated it to 0.38.0a0